pom.xml

```diff
@@ -57,7 +57,7 @@
     <objenesis.version>3.1</objenesis.version>
     <parquet.version>1.12.3</parquet.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <shadeBase>net.snowflake.ingest.internal</shadeBase>
+    <shadeBase>net.snowflake.ingest.shade</shadeBase>
```

Comment:
I think `shade` makes the use more clear vs internal could be something else, ex. https://github.com/snowflakedb/snowflake-ingest-java/tree/master/src/main/java/net/snowflake/ingest/streaming/internal